main.tex

```diff
@@ -331,21 +331,21 @@ \subsection*{Genetic backgrounds carrying resistance alleles}
 
 %%
 %
-In our global analysis of the Ag1000G phase 1 resource \cite{Ag1000gConsortium2017}, we used 1710 biallelic SNPs from within the 73.5 kbp \textit{Vgsc} gene (1607 exonic, 103 intronic) to compute the number of SNP differences between all pairs of 1530 haplotypes derived from the 765 wild-caught mosquitoes in the phase 1 cohort.
+In our initial report of the Ag1000G phase 1 resource \cite{Ag1000gConsortium2017}, we used 1710 biallelic SNPs from within the 73.5 kbp \textit{Vgsc} gene (1607 intronic, 103 exonic) to compute the number of SNP differences between all pairs of 1530 haplotypes derived from 765 wild-caught mosquitoes.
 %
-We then used pairwise genetic distances to perform hierarchical clustering, and found that haplotypes carrying resistance alleles in codon \texttt{995} were grouped into 10 distinct clusters of near-identical haplotypes.
+We then used pairwise genetic distances to perform hierarchical clustering, and found that haplotypes carrying resistance alleles in codon \texttt{995} were grouped into 10 distinct clusters with near-identical haplotypes.
 %
 Five of these clusters carried the \texttt{L995F} allele (labelled F1-F5), and a further five clusters carried \texttt{L995S} (labelled S1-S5).
 %
-To confirm these initial findings, we used the same haplotype data to construct median-joining networks (Figure \ref{fig:networks}).
+As an alternative approach, we used the same haplotype data to construct median-joining networks (Figure \ref{fig:networks}).
 %
-The network analysis is similar to hierarchical clustering, but allows for the reconstruction and placement of intermediate haplotypes that may not be observed in the data. 
+The network analysis allows for the reconstruction and placement of intermediate haplotypes that may not be observed in the data. 
 %
 It also allows for non-hierarchical relationships between haplotypes, which may arise if recombination events have occured between haplotypes.
 %
-Furthermore, the visualisation of these networks allows relationships among closely-related haplotypes to be discerned.
+Furthermore, the networks allow relationships among closely-related haplotypes to be discerned.
 %
-We constructed these networks up to a maximum edge distance of 2 SNP differences, to ensure that each connected component in the resulting networks captures a collection of closely-related haplotypes.
+We constructed these networks up to a maximum edge distance of 2 SNP differences, to ensure that each connected component in the resulting networks captures a collection of closely-related haplotypes (Figure \ref{fig:networks}).
```

Comment:
The networks figure has already been cited in the sentence above, not sure another citation is needed here.